pydis_site/apps/api/models/bot/aoc_completionist_block.py

```diff
@@ -0,0 +1,21 @@
+from django.db import models
+
+from pydis_site.apps.api.models.bot.user import User
+from pydis_site.apps.api.models.mixins import ModelReprMixin
+
+
+class AocCompletionistBlock(ModelReprMixin, models.Model):
+    """A Discord user blocked from getting the AoC completionist Role."""
+
+    user = models.ForeignKey(
+        User,
+        on_delete=models.CASCADE,
+        help_text="The user that is blocked from getting the AoC Completionist Role"
+    )
```

Comment:
This should probably be a `OneToOneField` instead. A many `AocCompletionistBlock`-to-one `User` relationship does not make sense since each user can only be blocked once.

Reply:
Oh, yes. I didn't catch it. Thanks!